src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory_Methods.cs

```diff
@@ -36,14 +37,49 @@ internal IArgument CreateArgumentOperation(ArgumentKind kind, IParameterSymbol p
         {
             var value = Create(expression);
 
+            // put argument syntax to argument operation
+            var argumentExist = value.Syntax?.Parent is ArgumentSyntax;
+
+            // if argument syntax doesn't exist, this operation is implicit
             return new CSharpArgument(kind,
                 parameter,
                 value,
                 semanticModel: _semanticModel,
-                syntax: value.Syntax,
+                syntax: argumentExist ? value.Syntax.Parent : value.Syntax,
                 type: value.Type,
                 constantValue: default,
-                isImplicit: expression.WasCompilerGenerated);
+                isImplicit: expression.WasCompilerGenerated || !argumentExist);
+        }
+
+        private IVariableDeclaration CreateVariableDeclaration(BoundLocalDeclaration boundLocalDeclaration, SyntaxNode syntax)
+        {
+            return OperationFactory.CreateVariableDeclaration(boundLocalDeclaration.LocalSymbol, Create(boundLocalDeclaration.InitializerOpt), _semanticModel, syntax);
+        }
+
+        private IOperation CreateBoundCallInstanceOperation(BoundCall boundCall)
+        {
+            if (boundCall.Method == null || boundCall.Method.IsStatic)
+            {
```

Comment:
Is this check necessary? In error cases, this will drop the receiver when the method is static but a receiver was provided, and will not prevent `Create` from being called for instance methods when a receiver was not included.

Reply:
I don't know. I converted existing code to the method.
' Lazy<IOperation> instance = new Lazy<IOperation>(() => Create(((object)boundCall.Method == null || boundCall.Method.IsStatic) ? null : boundCall.ReceiverOpt));

tagging @dotnet/analyzer-ioperation for people who knows about this.

Reply:
I think @cston is correct, we probably don't want to drop receiver unless it is a bound type node. We should follow up on this with tests and necessary implementation changes.

Reply:
opened https://github.com/dotnet/roslyn/issues/22233